docs/docs/testing-your-assistant.mdx

```diff
@@ -117,113 +117,138 @@ Here are some examples:
+test cases using:
 
-```bash
-rasa test nlu --nlu train_test_split/test_data.yml
+```bash {2}
+rasa test nlu
+    --nlu train_test_split/test_data.yml
 ```
 
-If you don't want to create a separate test set, you can
-still estimate how well your model generalises using cross-validation.
-To do this, add the flag `--cross-validation`:
+To test your model more extensively, use cross-validation, which automatically creates
+multiple train/test splits:
 
-```bash
-rasa test nlu --nlu data/nlu.yml --cross-validation
+```bash {3}
+rasa test nlu
+    --nlu data/nlu.yml
+    --cross-validation
 ```
 
 You can find the full list of options in the
 [CLI documentation on `rasa test`](command-line-interface.mdx#rasa-test).
 
+:::tip hyperparameter tuning
+To further improve your model check out this
+[tutorial on hyperparameter tuning](https://blog.rasa.com/rasa-nlu-in-depth-part-3-hyperparameters/).
+:::
+
 
 ### Comparing NLU Pipelines
 
-By passing multiple pipeline configurations (or a folder containing them) to the CLI, Rasa will run
-a comparison between the pipelines:
+To get the most out of your training data, you should train and evaluate your model on different pipelines
+and different amounts of training data.
 
-```bash
-rasa test nlu --config config_1.yml config_2.yml
-  --nlu data/nlu.yml --runs 3 --percentages 0 25 50 70 90
+To do so, use:
+
+```bash {2}
+rasa test nlu --nlu data/nlu.yml
+   --config config_1.yml config_2.yml
 ```
 
-The command in the example above will create a train/test split from your data,
-then train each pipeline multiple times with 0, 25, 50, 70 and 90% of your intent data excluded from the training set.
-The models are then evaluated on the test set and the f1-score for each exclusion percentage is recorded. This process
-runs three times (i.e. with 3 test sets in total) and then a graph is plotted using the means and standard deviations of
-the f1-scores.
+This performs several steps:
+1. Create a global 80% train / 20% test split from `data/nlu.yml`.
+2. Exclude a certain percentage of data from the global train split.
+3. Train models for each configuration on remaining training data.
+4. Evaluate each model on the global test split.
+
+The above process is repeated with different percentages of training data excluded in step 2 above.
+Additionally, the whole process is repeated three times for each configuration specified.
+A graph with the mean and standard deviations of
+[f1-scores](https://scikit-learn.org/stable/modules/generated/sklearn.metrics.f1_score.html)
+across all runs is plotted.
 
-The f1-score graph, along with all train/test sets, the trained models, classification and error reports, 
+The f1-score graph, along with all train/test sets, the trained models, classification and error reports,
 will be saved into a folder called `nlu_comparison_results`.
 
-### Intent Classification
+Inspecting the f1-score graph can help in understanding if you have enough data for your NLU model.
```

Comment:
Add: "If the graph shows that f1 is still improving when all of the training data is used, it may improve further with more data. But if f1 is flat when all training data is used, adding more data may not help."